fix(docs): normalize table headings (#5227)
hammerandchisel/discord-api-docs: docs/interactions/Message_Components.md

> 

Reviewer: ```suggestion

All of these styles require the `custom_id` field to be present except for `LINK`
```

> 

Reviewer: ```suggestion

All of these styles require the `custom_id` field to be present except for `LINK`, which requires `url`.

```

###### Button Styles

> | Name      | Value | Description                                       |

Reviewer 2: Removing columns like these just ruins human readability.

| --------- | ----- | ------------------------------------------------- |
| Primary   | 1     | Color `blurple`. Requires `custom_id`             |
| Secondary | 2     | Color `grey`. Requires `custom_id`                |
| Success   | 3     | Color `green`. Requires `custom_id`               |
| Danger    | 4     | Color `red`. Requires `custom_id`                 |

> | Link      | 5     | Color `grey`, navigates to a URL. Requires `url`  |

Reviewer: ```suggestion
| Primary   | 1     | Color `blurple`.             |
| Secondary | 2     | Color `grey`.                |
| Success   | 3     | Color `green`.               |
| Danger    | 4     | Color `red`.                 |
| Link      | 5     | Color `grey`, navigates to a URL when clicked  |
```
